### ISSUE TITLE
Vampire Weirdness

### ISSUE THREAD

**Issue by Kurfursten** [d3athrow/vgstation13#21932]
A player reported becoming a vampire midround out of the blue with no message. Even though they created (and had attached to role) clan, they appeared as an orphaned role on check antags.

One other vampire clan (Bumblefuck) had no members at all and there were no destroyed vampires at this point, so I don't know how this empty clan was created.

**Comment by ShiftyRail** [comment]
Runtimes ?

**Comment by Kurfursten** [comment]
Didn't see any that seemed related but it was hard to parse so late after the fact

**Comment by Kurfursten** [comment]
Have never seen this happen again
